### PR TITLE
chore: remove self hosted runner try 2 

### PR DESCRIPTION
**Pull request by schristoff**

### CODE REVIEW

File: .github/workflows/build_pipelinesrelease_template.yml [hunk 1]
```diff
@@ -124,7 +124,7 @@ jobs:
     needs:
     - Validate-xbuild
     runs-on:
-      - self-hosted
+      - ubunut-latest
```
Comment (troy0820) [inline .github/workflows/build_pipelinesrelease_template.yml:127]: You may have to unindent this.

**Review by troy0820 — CHANGES_REQUESTED** [review]
You may have to put this on the line or unindent this.

File: .github/workflows/build_pipelinesrelease_template.yml (hunk 1 above)
Comment (troy0820) [reply .github/workflows/build_pipelinesrelease_template.yml:127]:
```suggestion
    - ubunut-latest
```

**Comment by schristoff** [comment]
you got it in one of your PRs 🫡

**Pull request closed without merging**